releasenotes/notes/31797.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: enhancement
+area: environments
+issue:
+- 31732
+releaseNotes:
+- |
+  **Improved** Allow a user to use slash characters in environment variable values.
```

Comment:
`**Improved** Allow ...` is not grammatically correct.  Maybe this could be written as:
`**Added** support for slash characters in environment variables on injectionURL`

I'm  not sure if this function is used elsewhere.